src/node.cc

```diff
@@ -2844,6 +2845,11 @@ void SetupProcessObject(Environment* env,
     // after LoadEnvironment() has run.
   }
 
+  // --track-heap-objects
+  if (track_heap_objects) {
+    READONLY_PROPERTY(process, "trackHeapObjects", True(env->isolate()));
+  }
```

Comment:
Does it even need to be exposed to JS land?

(I guess the same question applies to `.traceSyncIO`, /cc @trevnorris.)